core/integer/digits_spec.rb

```diff
@@ -0,0 +1,34 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+ruby_version_is "2.4" do
+  describe "Integer#digits" do
+    it "returns an array of place values in base-10 by default" do
+      12345.digits.should == [5,4,3,2,1]
+    end
+
+    it "returns digits by place value of a given radix" do
+      12345.digits(7).should == [4,6,6,0,5]
+    end
+
+    it "allows decimals for the radix" do
+      12345.digits(2.5).should == [1, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 0, 1, 1]
```

Comment:
Could you replace 2.5 with `mock_int(2)` and mention #digits converts the radix with #to_int in the `it` description?